src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SPLIT_BY = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private String line = "";
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supplyCount = 0;
+        int buyCount = 0;
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(fromFileName));
```

Comment:
not informative naming(br, stringBuilder, fileWriter), how to guess what it stores?